examples/util/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
Surely we want to not gitignore the other existing files in examples too?

Reply:
Opps, this was supposed to be in `examples/util/v1`